src/utils/tana-formatter/content-processing.ts

```
@@ -0,0 +1,477 @@
+function isEmptyBulletNode(line: string): boolean {
+  const trimmed = line.trim()
+  return (
+    trimmed === '-' ||
+    trimmed === '•' ||
+    trimmed === '*' ||
+    trimmed === '- •' ||
+    trimmed === '- *' ||
+    trimmed === '-•' ||
+    trimmed === '-*' ||
+    /^-\s*[•*\u200B\u200C\u200D\uFEFF]*\s*$/.test(trimmed) ||
+    /^[-\s\u200B\u200C\u200D\uFEFF]*$/.test(trimmed)
```

Biome (1.9.4)

[error] A character class cannot match a joined character sequence.

A zero width joiner composes several emojis into a new one. Replace the character class with an alternation.

(lint/suspicious/noMisleadingCharacterClass)